wolfcrypt/src/pkcs12.c

```diff
@@ -242,7 +242,7 @@ static int GetSafeContent(WC_PKCS12* pkcs12, const byte* input,
         freeSafe(safe, pkcs12->heap);
         return ASN_PARSE_E;
     }
-    if (GetLength(input, &localIdx, &size, maxIdx) <= 0) {
+    if (GetLength(input, &localIdx, (int *)&size, maxIdx) <= 0) {
```

Comment:
Can't do this.
int is not always 32 bits.

Reply:
:heavy_check_mark: